webapp/src/components/DynamicButtonDataTable.vue

```diff
@@ -24,44 +24,64 @@
       </button>
       <button
         v-if="dataType === 'collections'"
-        class="btn btn-default"
+        class="btn btn-default ml-2"
         @click="$emit('open-create-collection-modal')"
       >
         Create new collection
       </button>
       <button
         v-if="dataType === 'startingMaterials' && editableInventory"
-        class="btn btn-default"
+        class="btn btn-default ml-2"
         @click="$emit('open-create-item-modal')"
       >
         Add a starting material
       </button>
       <button
         v-if="dataType === 'equipment'"
-        class="btn btn-default"
+        class="btn btn-default ml-2"
         @click="$emit('open-create-equipment-modal')"
       >
         Add an item
       </button>
     </div>
     <div class="button-right d-flex">
```

Comment:
I was confused about where this `button-right` class is defined, but I now see it seems to be on the DynamicButtonDataTable.vue

Ideally, we would use `<style scoped>` on each component and the styles from one component that way wouldn't affect the others. There is a way to use "deep" selectors on a scoped component which is occasionally needed, but in general having the styles defined in the component where they are used is best.

Reply:
Yeah, I must have forgotten to move it when the buttons didn't have their own components yet, thansks!